src/Commands/Env/CloneContentCommand.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Pantheon\Terminus\Commands\Env;
+
+use Pantheon\Terminus\Commands\CloneCommand;
+
+class CloneContentCommand extends CloneCommand
+{
+
+    /**
+     * Clones content from one environment to another.
+     *
+     * @command env:clone-content
+     *
+     * @usage terminus env:clone foo.live foo.dev
+     *   Clones the files and database from the live environment of the 'foo'
+     *   site to the development environment of the 'foo' site.
+     *
+     * @param $origin
+     * @param $target
+     *
+     */
+    public function cloneContent($origin, $target)
+    {
+        $this->invokeClone($origin, $target, ['database', 'files']);
```

Comment:
Can you elaborate? This was done this way so that we can later do commands like `db:clone` which will wrap the the higher level command and pass in only the databse. This was based on the discussion around usage I brought up with @ari-gold. I'm curious why this would be a bad pattern.